functions/kubernetes/kubernetes.md

```diff
@@ -3,26 +3,42 @@
 ### Prerequisites:
 -	Clone the https://github.com/dhaval055/shuffle repository using Git and change the branch to shuffle-k8s then navigate to the functions/kubernetes directory, which contains all the necessary Kubernetes configuration files for deployment.
 -	Running Kubernetes cluster.
-- Ensure you have a local Docker registry set up to store and manage Docker images for applications built with Shuffle. While the registry is crucial for handling custom-built apps, you’ll still be able to run workflows without it. 
-
+- Ensure you have a local Docker registry set up to store and manage Docker images for applications built with Shuffle. While the registry is crucial for handling custom-built apps, you’ll still be able to run workflows without it. To setup a docker registry, if you have docker installed on one of your node run following commands.
```

Comment:
leaving here for now, assumption is: the customer gets a guide to copy paste commands from. i know it's annoying but add in: `mkdir certs` as well.

if you don't, i will do it later so don't worry.